### PR TITLE
add radiobox.css@1.0.1 w/ npm auto-update

### LINKED ISSUE #8969: [Request] Add Radiobox.css
**Library name: Radiobox.css** 
**Git repository url: https://github.com/720kb/radiobox.css**
**npm package url(optional):** 
**License(s): MIT**
**Official homepage: https://720kb.github.io/radiobox.css**
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster than just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by nolsherry**
PR for #8969 
@pvnr0082t , @kennynaoh , please help me review this PR, thanks!
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/720kb/radiobox.css
 * Official website (optional, not the repository): https://720kb.github.io/radiobox.css
 * NPM package url (optional): https://www.npmjs.com/package/radiobox.css
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 609
   - Count of watchers: 14
   - Count of forks: 32
 * NPM download stats (optional): 
   - Downloads in the last day: 3
   - Downloads in the last week: 7
   - Downloads in the last month: 98

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [X] This lib was not found on cdnjs repo
 * [X] No already exist / duplicated issue and PR
 * [X] The lib has notable popularity
   * [X] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [X] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [X] Has valid tags for each versions (for git auto-update)
 * [X] Auto-update setup
 * [X] Auto-update target/source is valid.
 * [X] Auto-update filemap is correct.

# Git commit checklist
 * [X] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [X] The parent of the commit(s) in the PR is not old than 3 days.
 * [X] Pull request is sending from a non-master branch with meaningful name.
 * [X] Separate unrelated changes into different commits.
 * [X] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [X] Close corresponding issue in commit message
 * [X] Mention related issue(s), people in commit message, comment.

close #8969, cc @45kb

### CODE REVIEW

**Comment by pvnr0082t** [comment]
What about other css files under `css` folder?

**Comment by nolsherry** [comment]
@pvnr0082t, I have asked the author in [Issue6](https://github.com/720kb/radiobox.css/issues/6), and he  response that the only file needed is `dist/css/radiobox.min.css`, so I thick it enough to add  `dist/css/radiobox.min.css`.

**Comment by 45kb** [comment]
Hi i am the author, yes the only file you should include is the `dist/css/radiobox.min.css`

Then if you can serve all the other files inside `dist/css` could be better and appreciated a lot.

Thank you

**Comment by PeterDaveHello** [comment]
@45kb I'll like to serve them all, thanks for your comment!

**Comment by pvnr0082t** [comment]
@nolsherry I suggest you just paste the link like this -> https://github.com/720kb/radiobox.css/issues/6 . It will be more clear.

**Comment by nolsherry** [comment]
@45kb , I will serve them all, thanks for your response!

cc @PeterDaveHello @pvnr0082t

**Comment by 45kb** [comment]
Thank you a lot, appreciated a lot!

**Comment by PeterDaveHello** [comment]
@nolsherry let's finish this PR ASAP!
@45kb Thanks for your help and thanks for your patient!

**Comment by nolsherry** [comment]
@PeterDaveHello , no problem!
@pvnr0082t , @kennynaoh , I have modified the filemap in `package.json`  that can serve all the other files inside `dist/css`, please help me review this PR, thanks!

File: ajax/libs/radiobox.css/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  },
+  "author": "Filippo Oretti <filippo.oretti@gmail.com>",
+  "license": "MIT",
+  "homepage": "https://720kb.github.io/radiobox.css",
+  "npmName": "radiobox.css",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "css/*"
```
Comment (pvnr0082t) [inline ajax/libs/radiobox.css/package.json]: @nolsherry I think it's better to use `"!(scss)/*"` here.

**Comment by nolsherry** [comment]
@pvnr0082t , @x09326 , I have modified the `filemap` in `package.json` into `"!(scss)/*"`, please help me review this PR, thanks!

**Comment by pvnr0082t** [comment]
LGTM

**Comment by kennynaoh** [comment]
LGTM

**Pull request merged**